app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/SceneService.scala

```diff
@@ -36,21 +67,59 @@ class SceneService[ProjStore: ProjectStore, HistStore](
       ForeignError.handle {
         AuthedService {
           case GET -> Root / UUIDWrapper(sceneId) / IntVar(z) / IntVar(x) / IntVar(
-                y)
-                :? BandOverrideQueryParamDecoder(bandOverride) as user =>
+                y) as user =>
             val bbox = TileUtils.getTileBounds(z, x, y)
-            val eval =
-              LayerTms.identity(
-                scenes.read(sceneId, Some(bbox), bandOverride, None))
             for {
-              fiberAuth <- authorizers.authScene(user, sceneId).start
-              fiberResp <- eval(z, x, y).start
-              _ <- fiberAuth.join.handleErrorWith { error =>
-                fiberResp.cancel *> IO.raiseError(error)
+              authFiber <- authorizers.authScene(user, sceneId).start
+              bandsFiber <- getDefaultSceneBands(sceneId)
+              footprintFiber <- getSceneFootprint(sceneId)
+              _ <- authFiber.join.handleErrorWith { error =>
+                bandsFiber.cancel *> footprintFiber.cancel *> IO.raiseError(
+                  error)
+              }
+              footprint <- footprintFiber.join
```

Comment:
I don't think this is used anywhere here

Reply:
Removed